Revert the revert and try again

aeslaughter opened the pull request:
<!--
INCLUDE THE FOLLOWING IN THE PR DESCRIPTION
- Explain relevant design information for your change.
- Follow the [Coding Standards](http://mooseframework.org/wiki/CodeStandards/).
- Submit or improve [Test Cases](http://mooseframework.org/wiki/MooseTraining/testing/).
- Reference a specific issue, place "refs #<issue>" or "closes #<issue>" (e.g., #closes #1234).
-->

aeslaughter committed:
Further improvments to Exodus tab logic

aeslaughter committed:
Revert "Merge pull request #11739 from aeslaughter/peacock-revert"

This reverts commit 27f31542aae3a21a6e00da9657e6d5a9736f8c73, reversing
changes made to 9a0b375bdae0a80082e4d097eefd0f08935e5e31.

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/204711/) on bcadcaf wanted to post the following:

View the site [here](http://mooseframework.inl.gov/docs/PRs/11748/site)

This comment will be updated on new commits.

Commit:
Minor changes for linux test

aeslaughter committed:
Allow variable list to be modified with arrow keys

aeslaughter committed:
Add a loading message at startup

aeslaughter committed:
Create test for Extents load/store problem

(refs #11756)

This problem is beyond the scope of this current PR, it is also a fairly obscure problem that does not break Peacock. I will address this in another PR

aeslaughter committed:
Remove initial image test, this is already handled by the main window

aeslaughter commented:
@permcody @friedmud Can one or both of you give this a test drive.

permcody reviewed (approved):
Seems much more stable. We'll run with this version and see how things go.
